nservicebus/sqlserver/transactions_native_sqlserver_[3,).partial.md

```diff
@@ -0,0 +1 @@
+There was no distinction between `ReceiveOnly` and `SendsAtomicWithReceive`. Using native transaction was equivalent to `SendsAtomicWithReceive` mode.
```

Comment:
I'd say :

"Using native transaction is equivalent to [`SendsAtomicWithReceive` mode in Version 3 and higher](/nservicebus/sqlserver/transactions.md?version=sqlserver_3#native-transactions).

Note the link above might be wrong, I made it up in my head :)